modules/stochastic_tools/src/samplers/Cartesian1DSampler.C

```diff
@@ -0,0 +1,77 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "Cartesian1DSampler.h"
+
+registerMooseObjectAliased("StochasticToolsApp", Cartesian1DSampler, "Cartesian1D");
+
+InputParameters
+Cartesian1DSampler::validParams()
+{
+  InputParameters params = Sampler::validParams();
+  params.addClassDescription("Provides complete Cartesian product for the supplied variables.");
+  params.addRequiredParam<std::vector<Real>>(
+      "linear_space_items",
+      "A list of triplets, each item should include the min, step size, and number of steps.");
```

Comment:
I thought it was min step_size and divisions?

Reply:
It's number of steps. So one step will just return the min value, two steps would return the min  and min + step size, etc.

Reply:
This is the same syntax as `CartesianProduct`

Reply:
I think it s not consistent with the code?

Reply:
nevermind